packages/react-router/modules/__tests__/StaticRouter-test.js

```diff
@@ -31,18 +27,10 @@ describe("A <StaticRouter>", () => {
 
   describe("context", () => {
     let context;
-    class ContextChecker extends React.Component {
```

Comment:
If you don't mind I think I'll revert this test change, @pshrmn. I just wanna test the raw context value coming out of the `Consumer` instead of `withRouter`. It's a dumb test because it tests private API, but I wanted to make sure we had tests around the old context API and the new one to make sure they both worked.

Reply:
Oh wait, I understand now why you made this change. Because we don't export `RouterContext` in `react-router`.